motionplan/motionPlanner.go

```diff
@@ -115,12 +115,15 @@ func (req *PlanRequest) validatePlanRequest() error {
 			return errors.Errorf("%s does not have a start configuration", req.Frame.Name())
 		}
 		if frameDOF != len(seedMap) {
-			return frame.NewIncorrectDoFError(len(seedMap), len(req.Frame.DoF()))
+			return referenceframe.NewIncorrectDoFError(len(seedMap), len(req.Frame.DoF()))
 		}
 	} else if ok && frameDOF != len(seedMap) {
-		return frame.NewIncorrectDoFError(len(seedMap), len(req.Frame.DoF()))
+		return referenceframe.NewIncorrectDoFError(len(seedMap), len(req.Frame.DoF()))
 	}
 
+	// TODO: This should replace Frame and Goal
+	req.allGoals = map[string]*referenceframe.PoseInFrame{req.Frame.Name(): req.Goal}
```

Comment:
Suppose I have two arms and would like to perform coordinated motion. Shouldn't we specify the goals for both frames?

Reply:
There remains no way to currently express that using a PlanRequest, so there's no need to do so now.